go.mod

```diff
@@ -5,26 +5,34 @@ go 1.13
 require (
 	github.com/aws/aws-sdk-go v1.16.26
 	github.com/banzaicloud/k8s-objectmatcher v1.1.0
-	github.com/coreos/pkg v0.0.0-20180108230652-97fdf19511ea
+	github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f
 	github.com/coreos/prometheus-operator v0.34.0
 	github.com/corpix/uarand v0.1.1 // indirect
 	github.com/davecgh/go-spew v1.1.1
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-ini/ini v1.51.1
 	github.com/go-sql-driver/mysql v1.4.1
-	github.com/google/go-cmp v0.3.1
+	github.com/google/go-cmp v0.4.0
+	github.com/google/gofuzz v1.1.0 // indirect
 	github.com/google/uuid v1.1.1
 	github.com/icrowley/fake v0.0.0-20180203215853-4178557ae428
 	github.com/k8snetworkplumbingwg/network-attachment-definition-client v0.0.0-20200401090632-ee27f62faef8
-	github.com/kube-object-storage/lib-bucket-provisioner v0.0.0-20200610144127-e2eec875d6d1
+	github.com/kube-object-storage/lib-bucket-provisioner v0.0.0-20200107223247-51020689f1fb
```

Comment:
This change reverts the version of `lib-bucket-provisioner` to an older commit from back in January.  Can you remove this change, run `go mod tidy` and rebuild? I don't think we want to change this library version to an older version unless there's a good reason that we understand.

Reply:
seem it was conflict with this commit https://github.com/rook/rook/commit/e4fae54e3c24917e437f6fe850cea8b8b1fe2a03 before and i missed when rebasing my commit. fixed now. thanks @jbw976

Reply:
ah, thanks for fixing that!